### DISCUSSION TITLE
Printed circuit boards for WeMos D1 Mini

### DISCUSSION THREAD

**Discussion by phord** [phord/Jarvis#18] [Show and tell]
Keeping in mind that I'm an idiot, I still managed to design, draw and order printed circuit boards. Also keep in mind that these are the first custom PCB's I've ever ordered. I designed one for the WeMos D1 Mini and another for the NodeMCU.  
![image](https://user-images.githubusercontent.com/123908/153550772-645da848-67be-4977-a201-a053d954a8a4.png)

They arrived today. I'm glad I ordered two because the ones for the NodeMCU have a bad footprint. The through holes are too small. It's not really my fault, but I should have checked closer somehow. Anyway, the Wemos D1s fit perfectly.  

![image](https://user-images.githubusercontent.com/123908/153551009-0b0c8c52-ab72-4954-b93b-04476bcd1641.png)
![image](https://user-images.githubusercontent.com/123908/153551021-e546963d-fde0-46b6-988c-74aa81435be2.png)

I changed the pinouts to make the routing cleaner.  So there is a code change to work on these boards. But after that, it worked on the first try.  And it looks a bit cleaner, now.

I have some circuitboard drill bits somewhere, so I'll try to drill the NodeMCU holes wider later on.

**Comment by nikandriko** [comment]
Phil, these are awesome, mine arrived today. 
I used RJ-45 connectors from Amazon which were metallic and I thought looked a bit better.
I've velcroed it with 3M hanging strips underneath the desk and it looks awesome as the led light blinks when it receives the commands from the internet.
![image](https://user-images.githubusercontent.com/962911/156494283-50ac16da-3578-4e11-93d1-887af0b9239a.png)
